app/controllers/api/v0/packages_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Api
+  module V0
+    class PackagesController < ApplicationController
+      def show
+        sql = 'lower(package_name) like ?'
+        name = "%#{params[:id]}%"
+        @search = Package.where(sql, name)
+        render json: @search
```

Comment:
integrate serializer. It would be interesting to use this one https://github.com/Netflix/fast_jsonapi